lib/internal/Magento/Framework/App/Request/Http.php

```diff
@@ -228,8 +223,7 @@ public function getFrontName()
     public function setRouteName($route)
     {
         $this->route = $route;
-        $module = $this->routeConfig->getRouteFrontName($route);
-        if ($module) {
+        if ($module = $this->routeConfig->getRouteFrontName($route)) {
```

Comment:
Assignment in `if` should not be used.

Reply:
Thanks for this.  I will change my habit from this feedback.